docs/gateways/OnDemand.md

```diff
@@ -122,6 +122,70 @@ The _Interactive Applications_ menu contains options to launch certain applicati
 * Closing the window will not terminate the job, you can use the “My Interactive Sessions” tab to view all open interactive sessions and terminate them.
 * One can access a single GPU via the `Jupyter Session (Custom)` application by following the instructions provided in the [GPU access for Jupyter Sessions](#gpu-access-for-jupyter-sessions) section below. 
 
+###### Creating a Jupyter Session Conda Environment
+
+In Jupyter Session applications you have the option to launch a Jupyter session using a Conda environment that you have created. This becomes extremely useful if you are using a package that requires extensions be installed in the environment that is launching the Jupyter session. In order to correctly configure your environment so that it launches correctly, you need to ensure that the appropriate packages are installed in it. Below we provide the process needed to correctly create your environment using a Jupyter Session terminal. 
```

Comment:
change "In order to correctly configure" to "In order to configure" to avoid redundant use of "correctly".